CONTRIBUTING.md

```diff
@@ -60,6 +60,16 @@ make help
 
 ---
 
+## Local development using a container
+
+Horust was developed to be used inside containers. That means that it's primarily built to be used in *nix environments,
+and even more specifically in Linux and its flavours. At the time of writing, Horust fails to compile on Mac OSX due 
+to a dependency (`prctl`) that does not exist on Mac, as well as some `libc` functions that are Linux-sepcific and are
+not available on OSX. 
```

Comment:
I would not put it in these (kind of negative) terms. I would just say that if you don't want to install / setup cargo and the project's stuff (or you cannot get to build it because you're on Macos) then you can just go with the AIO docker container wrapper

Reply:
Fixed